site/specs/global.yml

```diff
@@ -514,8 +557,8 @@ info:
     |`PROPAGATION_EXCEPTION`|`315`|The server encountered an error while
     applying the configuration options. Some of the configuration might have
     been applied successfully. You can safely execute the configuration
-    operation again, if the problem persists please contact the Bandwidth
-    support team.|
+    operation again, if the problem persists please contact the voxbone support
```

Comment:
Keep voxbone? If yes, capitalize it. Suggest same for five more references below-:)